Resources/Prototypes/SimpleStation14/Loadouts/jobs.yml

```diff
@@ -241,6 +252,7 @@
   exclusive: true
   jobWhitelist:
     - StationEngineer
+    - ChiefEngineer
```

Comment:
Same as CMO, I don't feel the CE needs to be able to select this, as their role isn't simply to be an Engineer, it's to be a leader and Command.

```suggestion
```